Assets/Scripts/Events/EventQueue.cs

```diff
@@ -218,12 +218,14 @@ public async Task Load(EventQueueDetails details)
             foreach (string eventName in details.headliners ?? new List<string>())
             {
                 Event e = await Addressables.LoadAssetAsync<Event>(eventName).Task;
+                if (e == null) continue;
                 _headliners.AddLast(e);
             }
```

Comment:
do these need to be a comparison to null or can they just
```suggestion
            {
                Event e = await Addressables.LoadAssetAsync<Event>(eventName).Task;
                if (e) _headliners.AddLast(e);
            }
```

Reply:
I'm gonna keep it an explicit check, just to be safe

Reply:
null check is usually expensive compared to just checking truthiness